### ISSUE TITLE
Make numbers searchable by name

### ISSUE THREAD

**Issue by timonegk** [signdict/website#866]
Currently, the numbers are only found when searching for the number in digits. I tend to search numbers by writing them out and it would be nice if they could be found under the word too.

**Comment by jbaugh** [comment]
I am new to this project, so forgive me if this does not make sense. Would it make sense to (manually) change the entry's `text` column for each of the numbers to be something like `2 (zwei)`, `21 (einundzwanzig)`, etc. My German is quite poor, so there might be problems with this approach.

Alternatively, expanding the `entries` table might make sense to go with long term, because you can include things like synonyms. Let me know your thoughts and I'll see if I can make any progress.

**Comment by bitboxer** [comment]
Yes, this is also a valid approach, but I would prefer the synonyms way. It's more flexible in the long run and I already needed it in a couple of places.
